kion/resource_label_test.go

```diff
@@ -0,0 +1,162 @@
+package kion
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	hc "github.com/kionsoftware/terraform-provider-kion/kion/internal/kionclient"
+)
+
+const (
+	resourceTypeLabel = "kion_label"
+	resourceNameLabel = "label1"
+)
+
+var (
+	initialTestLabel = hc.LabelCreate{
+		Color: "#123abc",
+		Key:   "environment",
+		Value: "production",
+	}
+	updatedTestLabel = hc.LabelCreate{
+		Color: "#abcdef",
+		Key:   "env",
+		Value: "prod",
+	}
+)
+
+func TestAccResourceLabelCreate(t *testing.T) {
+	config := testAccLabelGenerateResourceDeclaration(&initialTestLabel)
+	hc.PrintHCLConfig(config) // Print the generated HCL configuration
```

Comment:
Unit tests are usually suppose to be pretty quiet, except for on failures.  This is going to make them pretty noisy with non-useful information when running all tests at once.  Should this be commented by default, making it easy to uncomment when needed?

Reply:
Good point.  I did it to see the test results because I didn't know exactly what to expect and what it was looking for.  I'll comment it out.